src/data/localization.rs

```diff
@@ -497,10 +518,37 @@ impl Localization {
         }
     }
 
+    #[cfg(feature = "ck3")]
+    fn check_collisions(&self, lang: &str) {
+        for (k, v) in self.all_collision_keys(lang) {
+            let mut keys_str: String = String::from("[");
```

Comment:
In tbe helpers module there is a `stringify_list` function that does this in a more human-readable way